doc/src/devdocs/functions.md

```diff
@@ -101,10 +101,8 @@ currently share a method table via special arrangement.
 
 The "builtin" functions, defined in the `Core` module, are:
 
-```
-=== typeof sizeof <: isa typeassert throw tuple getfield setfield! fieldtype
-nfields isdefined arrayref arrayset arraysize applicable invoke apply_type _apply
-_expr svec
+```@eval
+[n for n in names(Core;all=true) if getfield(Core,n) isa Core.Builtin]
```

Comment:
This will render as
```
<:===_apply_apply_latest_apply_pure_expr_typevarapplicableapply_typearrayrefarraysetarraysizeconst_arrayreffieldtypegetfieldgetpropertyifelseinvokeisaisdefinednfieldssetfield!setproperty!sizeofsvecthrowtupletypeasserttypeof
```